sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Azure.Monitor.OpenTelemetry.Exporter.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>An OpenTelemetry .NET exporter that exports to Azure Monitor</Description>
     <AssemblyTitle>AzureMonitor OpenTelemetry Exporter</AssemblyTitle>
-    <Version>1.0.0-beta.11</Version>
+    <Version>1.0.0-beta.12</Version>
```

Comment:
Same as above, since 1.0.0-beta.11 is not released at all, we should not change this version number. Otherwise we are skipping the version 1.0.0-beta.11

Reply:
Didn't realize beta.11 wasn't out yet.  Rolling back as requested.